browser-ext/content/content.js

```
@@ -0,0 +1,361 @@
+            case 'url':
+                element.value = value;
+                break;
+
+            case 'textarea':
+                element.value = value;
+                break;
+
+            case 'select-one':
+                const option = Array.from(element.options)
+                .find(opt =>
+                    opt.text.toLowerCase().includes(value.toLowerCase()) || opt.value.toLowerCase().includes(value.toLowerCase()));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)